mathics/builtin/quantum_mechanics/angular.py

```diff
@@ -0,0 +1,220 @@
+"""
+Angular Momentum
+
+<url>:Angular momentum: https://en.wikipedia.org/wiki/Angular_momentum</url> in physics is the rotational analog of linear momentum. It is an important quantity in physics because it is a conserved quantity the total angular momentum of a closed system remains constant.
+"""
+
+from typing import List
+
+from sympy.physics.matrices import msigma
+from sympy.physics.quantum.cg import CG
+from sympy.physics.wigner import wigner_3j, wigner_6j
+
+from mathics.builtin.base import SympyFunction
+from mathics.core.atoms import Integer
+from mathics.core.convert.sympy import from_sympy
+from mathics.core.evaluation import Evaluation
+from mathics.core.list import ListExpression
+from mathics.core.symbols import Symbol
+
+from mathics.core.attributes import (
+    listable as A_LISTABLE,
+    # numeric_function as A_NUMERIC_FUNCTION,
+    protected as A_PROTECTED,
+    read_protected as A_READ_PROTECTED,
+)
+
+
+class ClebschGordan(SympyFunction):
+    """
+    <url>:Clebsch-Gordan coefficients matrices: https://en.wikipedia.org/wiki/Clebsch%E2%80%93Gordan_coefficients</url> (<url>:SymPy: https://docs.sympy.org/latest/modules/physics/quantum/cg.html</url>, <url>:WMA: https://reference.wolfram.com/language/ref/ClebschGordan</url>)
+
+    <dl>
+      <dt>'ClebschGordan[{$j1$, $m1$}, {$j2$, $m2$}, {$j$ $m$}]'
+      <dd>returns the Clebsch-Gordan coefficient for the decomposition of in terms of |$j$, $m$|$j2$, $m2$>.
+    </dl>
+
+    >> ClebschGordan[{3 / 2, 3 / 2}, {1 / 2, -1 / 2}, {1, 1}]
+     = Sqrt[3] / 2
+
+    Compare with WMA example:
+    >> ClebschGordan[{5, 0}, {4, 0}, {1, 0}] == Sqrt[5 / 33]
+     = True
+
+    """
+
+    attributes = A_PROTECTED | A_READ_PROTECTED
+    summary_text = "Clebsch-Gordan coefficient"
+    sympy_name = "physics.quantum.cg.CG"
+
+    def apply(
+        self,
+        j1m1: ListExpression,
+        j2m2: ListExpression,
+        jm: ListExpression,
+        evaluation: Evaluation,
+    ):
+        "ClebschGordan[j1m1_List, j2m2_List, jm_List]"
+
+        sympy_jms: List[int] = []
+
+        for pair in (j1m1, j2m2, jm):
+            if len(pair.elements) != 2:
+                return
+            sympy_jms += [p.to_sympy() for p in pair.elements]
+        return from_sympy(CG(*sympy_jms).doit())
+
+
+class PauliMatrix(SympyFunction):
+    """
+    <url>:Pauli matrices: https://en.wikipedia.org/wiki/Pauli_matrices</url> (<url>:SymPy: https://docs.sympy.org/latest/modules/physics/matrices.html#sympy.physics.matrices.msigma</url>, <url>:WMA: https://reference.wolfram.com/language/ref/PauliMatrix.html</url>)
+
+    <dl>
+      <dt>'PauliMatrix[$k$]'
+      <dd>returns the $k$th Pauli spin matrix).
+    </dl>
+
+    >> PauliMatrix[Range[3]]
+     = {{{0, 1}, {1, 0}}, {{0, -I}, {I, 0}}, {{1, 0}, {0, -1}}}
+    """
+
+    attributes = A_LISTABLE | A_PROTECTED | A_READ_PROTECTED
```

Comment:
`PauliMatrix` is not `Listable` in WL. However, I have to say that this would be an improvement...

Reply:
Thanks. Should be addressed by 270738c7